academy/3-ibc/4-clients.md

```diff
@@ -175,7 +173,13 @@ The next validator set is used for verifying subsequent submitted headers or upd
 
 The root is the **AppHash**, or the hash of the application state of the counterparty blockchain that this client is representing. This root hash is particularly important because it is the root hash used on a receiving chain when verifying [Merkle](https://en.wikipedia.org/wiki/Merkle_tree) proofs associated with a packet coming over IBC, to determine whether or not the relevant transaction has been actually been executed on the sending chain. If the Merkle proof associated with a packet commitment delivered by a relayer successfully hashes up to this `ConsensusState` root hash, it is certain that the transaction was actually executed on the sending chain and included in the state of the sending blockchain.
 
-The following is an example of how the Tendermint client handles this Merkle [proof verification](https://github.com/cosmos/ibc-go/blob/main/modules/core/23-commitment/types/merkle.go). The [ICS-23 spec](https://github.com/cosmos/ibc/tree/master/spec/core/ics-023-vector-commitments) addresses how to construct membership proofs, and the [ICS-23 implementation](https://github.com/confio/ics23) currently supports Tendermint IAVL and simple Merkle proofs out of the box. Note that non-Tendermint client types may choose to handle proof verification differently:
+The following is an example of how the Tendermint client handles this Merkle [proof verification](https://github.com/cosmos/ibc-go/blob/main/modules/core/23-commitment/types/merkle.go). The [ICS-23 spec](https://github.com/cosmos/ibc/tree/master/spec/core/ics-023-vector-commitments) addresses how to construct membership proofs, and the [ICS-23 implementation](https://github.com/confio/ics23) currently supports Tendermint IAVL and simple Merkle proofs out of the box.
```

Comment:
I think IAVL is independent from Tendermint (see [here](https://github.com/cosmos/ics23/blob/master/go/proof.go#L9)) and [simple Merkle proofs](https://github.com/cosmos/ics23/blob/master/go/proof.go#L28) is the one that is [Tendermint specific](https://github.com/tendermint/tendermint/tree/main/crypto/merkle), I think.

Reply:
Not sure about this, but when you check out the [ICS-23 implementation](https://github.com/cosmos/ics23#supported-merkle-stores) lists:
- tendermint/iavl
- tendermint - SimpleMerkleProofs

Reply:
Ah, I see... But if you click on the link for `tendermint/iavl` then it goes to [cosmos/iavl](https://github.com/cosmos/iavl). Maybe the readme hasn't been updated for a long time and the IAVL repo was originally in the tendermint GitHub organization?